lib/rules/block-spacing.js

```diff
@@ -15,7 +15,8 @@ var util = require("../ast-utils");
 module.exports = function(context) {
     var always = (context.options[0] !== "never"),
         message = always ? "Requires a space" : "Unexpected space(s)",
-        sourceCode = context.getSourceCode();
+        sourceCode = context.getSourceCode(),
+        tokensAndComments = sourceCode.tokensAndComments;
```

Comment:
Is it part of the public API now?

Is there anything else we can do? I could do what @mysticatea proposes inside the rule. Or we could add a new property and deprecate this one.